lint/buf.bzl

```diff
@@ -87,14 +87,23 @@ def _buf_lint_aspect_impl(target, ctx):
     if not should_visit(ctx.rule, ctx.attr._rule_kinds):
         return []
 
-    report, exit_code, info = report_files(_MNEMONIC, target, ctx)
+    outputs, info = output_files(_MNEMONIC, target, ctx)
     buf_lint_action(
         ctx,
         ctx.toolchains[ctx.attr._buf_toolchain].cli,
         ctx.toolchains["@rules_proto//proto:toolchain_type"].proto.proto_compiler.executable,
         target,
-        report,
-        exit_code,
+        outputs.human.stdout,
+        outputs.human.exit_code,
+    )
+
+    buf_lint_action(
```

Comment:
same comment for buf lint as for shellcheck, https://github.com/aspect-build/rules_lint/pull/331/files#r1680423092